### ISSUE TITLE
Sending emails with Celery

### ISSUE THREAD

**Issue by AlekseiKhatkevich** [sunscrapers/djoser#500]
Hi
It s rather question then issue.

Is it any way to use Celery to send activation and other emails sent by Djoser.
Default implementation in Djoser  uses request  as Email Class init argument object to send emails and it's kind of non-serializable. Maybe you have an experience with making Celery send Djoser emails?

**Comment by vipin-s0106** [comment]
Hi,

This is issue still not resolved I can take this up.

**Comment by hvitis** [comment]
Any news? +1

**Comment by bazhenstvo** [comment]
The easiest option is to use [Django-Celery-Email](https://github.com/pmclanahan/django-celery-email), which would handle sending Djoser emails as well as any other emails.

**Comment by thelittlebug** [comment]
here is my https://github.com/sunscrapers/django-templated-mail/pull/29 that fixes this bug

**Comment by thelittlebug** [comment]
@bazhenstvo i do not believe its that simple because djoser is using an unpickleable email class, look at this issue: https://github.com/sunscrapers/django-templated-mail/issues/28
